src/lib/capability.ts

```diff
@@ -345,9 +345,9 @@ export class Capability implements CapabilityExport {
       return { ...commonChain, WithName, WithNameRegex };
     }
 
-    function InNamespaceRegex(...namespaces: RegExp[]): BindingWithName<T> {
+    function InNamespaceRegex(...namespaces: string[]): BindingWithName<T> {
```

Comment:
```suggestion
    function InNamespaceRegex(...namespaces: string[]): BindingWithName<T> {
```

This is a breaking change to the Pepr API that is in production and being used in delivery. I could be mistaken but i think we chatted about changing the regexes and decided against due to the need to serialize the bindings during the forking process

Reply:
That's right, we're serializing data from a string so we can't force the data to be a `RegExp` at the moment.